e2e-tests/server/chat.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+
+class Chat:
+    ALL = re.compile(r'^/all\s+(.+)$')
+    MSG = re.compile(r'^/msg\s+(\w+)\s+(.+)$')
+
+    def __init__(self, hub):
+        self._hub = hub
+        self._banned = set()
```

Comment:
this does not seem to be used

Reply:
Heh, I this is probably a leftover from the initial prototype where I had ambitions to build a bigger app. :slightly_smiling_face: